src/app/components/App.tsx

```diff
@@ -43,6 +43,20 @@ import pkg from '../../../package.json';
 import 'franklin-sites/franklin.css';
 import './styles/app.scss';
 
+// This is hackery is to prevent define being repeatedly called for the same
```

Comment:
I was curious to try without the hack and I didn't run into any problem of defining the custom element twice

Reply:
Oh that's interesting - no issue in the feature viewer tab either?

Reply:
When I switched from Feature viewer to overview/variant viewer tab, it happens and crashes too. It is needed after all